x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/with_security_context.tsx

```diff
@@ -51,8 +53,15 @@ export const withSecurityContext = <P extends {}>({
       store = createStore(
         combineReducers({
           management: managementReducer,
+          app: appReducer,
         }),
-        { management: undefined },
+        {
+          management: undefined,
+          // @ts-ignore ignore this error as we just need the enableExperimental and it's temporary
+          app: {
+            enableExperimental: ExperimentalFeaturesService.get(),
```

Comment:
Awesome 🤩 
like this, we can have our feature flags available to our component that are rendered outside of our plugin